### DISCUSSION TITLE
How to run Harvey OS on ARM?

### DISCUSSION THREAD

**Discussion by hmsjy2017** [Harvey-OS/harvey#1033] [Q&A] [answered]
I want to run Harvey OS on Raspberry Pi. Could you tell me how to do that? Thank you!

**Comment by orangecms** [comment, marked as answer]
Hi, so far there is a single `WFI` (wait for interrupt) instruction for ARM 64, see https://github.com/Harvey-OS/harvey/blob/main/sys/src/9/aarch64/l.S

I want to bring Harvey to ARM as well, would you like to contribute to that? :)
What I can tell you is that 9front supports the Raspberry Pi, see http://9front.org/releases

Feel free to join our Slack Team and let's have chat. :)
See https://harvey-os.org/community/
